doc/manual/manual.tex

```diff
@@ -8485,6 +8488,231 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+in a neighborhood of the interface, rather than in the entire computational domain. 
+
+\begin{figure}[b!]
+   \centering
+\includegraphics[width=0.5\textwidth]{cookbooks/benchmarks/vof-van-keken/rms_vel_comparison.png}
+   \caption{Evolution of the root mean square velocity as a function of time for computations 
+    of the Van Keken problem made with the VOF interface tracking algorithm with five 
+    different global mesh refinements.
+    Since the VOF initial conditions are discontinuous, the above results should really be 
+    compared with the computations with discontinuous initial conditions on the left in 
+    Figure~\ref{fig:vk-2}, although the above results also compare extremely favorably with the 
+    computations with smoothed, continuous initial conditions for the compositional field on 
+    the right in Figure~\ref{fig:vk-2}.
+    As in Figure~\ref{fig:vk-2}, 5 global refinements correspond to a $32 \times 32$ mesh and 10 global refinements correspond to a $1024 \times 1024$ mesh.}
+    \label{fig:vof-vk-1}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.4\textwidth]{cookbooks/benchmarks/vof-van-keken/vof_van_keken_refinement_comparison.png}
+    \caption{\it The results of two computations of the Van Keken problem made with the VOF 
+        interface tracking algorithm overlaid upon each other at $t_{end}=2000$.
+        This visualization consists of the \underline{two} zero contours of 
+        $\tilde{\psi}^7 \, (t_{end})$ and $\tilde{\psi}^8 \, (t_{end})$, where 
+        $\tilde{\psi}^7 \, (t_{end})$ is field where the zero contour is the reconstructed interface 
+        at the final time $t_{end}$ computed on a uniform grid with a refinement level of 7 and 
+        similarly for $\tilde{\psi}^8 \, (t_{end})$.
+        The color of the zero contour for $\tilde{\psi}^7 \, (t_{end})$ is black and the color of the 
+        zero contour $\tilde{\psi}^8 \, (t_{end})$ is bright green.
+        Compare with the fourth image on the right in Figure~\ref{fig:vk-1}.
+    }
+    \label{fig:VOF_van_Keken-02}
+\end{figure}
+
+Since, as noted above, when the interface is discontinuous, the van Keken problem is a version 
+of the Rayleigh-Taylor problem that is unstable to perturbations of all 
+wavelengths,\footnote{This is true whether the two fluids have the same viscosity or different 
+viscosities.} (e.g. see~\cite{SC:1961}). Therefore, it is extremely sensitive to the initial 
+conditions.
+Thus, rather than use the standard initialization procedure for the VOF method, which consists 
+of integrating the initial interface over each multi-material cell in which one material or 
+fluid has the value of $1$ and the other has the value of $0$; i.e., a characteristic function 
+representation of the interface~\cite{JMR:2019,JMR-EGP:2019}, instead we compute the initial volume fractions using a signed distance function $\phi$ as follows.
```

Comment:
Does this suggested edit preserve the necessary information, and make the sentence structure clearer? @egpuckett @naliboff 

```suggestion
In order to address this sensitivity, we do not use a standard composition quadrature initialization procedure
for the VOF method.
Instead we compute the initial volume fractions using a signed distance function $\phi$ as follows~\cite{JMR:2019,JMR-EGP:2019}.
```

Reply:
@class4kayaker - Yes, that rephrasing is much clearer, thanks!